contracts/ERC20.sol

```diff
@@ -2,25 +2,70 @@
 pragma solidity ^0.8.0;
 
 import "./IERC20.sol";
+import "./IERC20Metadata.sol";
+
+contract ERC20 is IERC20, IERC20Metadata {
+  uint256 public _totalSupply;
+  string public _name;
```

Comment:
When a variable / function is prefixed with _, most of the time it shall be internal/private variable